### PR TITLE
Tickets/dm 41082

### PR DESCRIPTION
**Pull request by dsanmartim**
Hi Tiago.

Here is a first try to the `sleep.py` script. I put it in the `maintel` folder.

Tks!

### CODE REVIEW

**Review by tribeiro — COMMENTED** [review]
Looks good.. My only comment is that this should probably be at the higher level, alongside `set_summary_state` etc, instead of being inside the `maintel` folder. 

Other than that I think it is good to go, just re-organize the commits and add the news fragment. Let me know and I can approve the PR.

File: python/lsst/ts/standardscripts/data/scripts/maintel/sleep.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+
+from lsst.ts.standardscripts.maintel import Sleep
```
Comment (tribeiro) [inline python/lsst/ts/standardscripts/data/scripts/maintel/sleep.py:25]: remove `maintel`

File: tests/test_sleep.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+
+from lsst.ts.standardscripts import BaseScriptTestCase, get_scripts_dir
+from lsst.ts.standardscripts.maintel.sleep import Sleep
```
Comment (tribeiro) [inline tests/test_sleep.py:25]: remove maintel

**Review by tribeiro — COMMENTED** [review]
Looking good.. See my inline comments.. the CI is failing because of that

File: tests/test_sleep.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        async with self.make_script():
+            sleep_for = 5
+            await self.configure_script(sleep_for=sleep_for)
+
+            # Run the script
+            await self.run_script()
+
+    async def test_executable(self):
+        scripts_dir = get_scripts_dir()
+        script_path = scripts_dir / "maintel" / "sleep.py"
```
Comment (tribeiro) [inline tests/test_sleep.py:55]: remove maintel

File: python/lsst/ts/standardscripts/data/scripts/maintel/sleep.py
Comment (tribeiro) [inline python/lsst/ts/standardscripts/data/scripts/maintel/sleep.py:1]: Please, move this to the directory above.

Comment (dsanmartim) [reply python/lsst/ts/standardscripts/data/scripts/maintel/sleep.py:1]: I think that now everything is fixed...

**Review by tribeiro — APPROVED** [review]
Looks good!

**Comment by dsanmartim** [comment]
Done!

**Pull request merged**